common/djangoapps/entitlements/api.py

```diff
@@ -0,0 +1,16 @@
+"""
+Python APIs exposed by the Entitlements app to other in-process apps.
+"""
+
+from .models import CourseEntitlement
```

Comment:
nit: import `CourseEntitlement` as `_CourseEntitlement` so people know not to import the model from the `api.py`. I think for this change, we don't need to go through creating the `models_api.py`, but you can see that suggestion in use here: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/models_api.py#L8